tracing-core/src/collect.rs

```diff
@@ -271,7 +271,7 @@ pub trait Collect: 'static {
     /// [visitor]: super::field::Visit
     /// [`record`]: super::span::Attributes::record
     /// [`record` method]: super::span::Record::record
-    fn record(&self, span: &span::Id, values: &span::Record<'_>);
+    fn record(&self, span: &span::Id, values: valuable::NamedValues<'_>);
```

Comment:
I think we should import it directly, but I don't care if we do it in a separate commit